hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1064,6 +1064,10 @@ private void stopSecretManager() {
     }
   }
 
+  public void refetchSecretKey() {
+    secretKeyClient.getCurrentSecretKey();
```

Comment:
This would not refresh the current key from SCM. You need to add a new method to `SecretKeySignerClient`, say `refetchCurrentSecretKey`, which should be implemented in `DefaultSecretKeySignerClient`. 
The implementation can look like follow:
```
 @Override
  public void refetchCurrentKey() {
    // pass duration as ZERO to force a refresh.
    checkAndRefresh(Duration.ZERO);
  }
```

The method `checkAndRefresh` should me made `synchronized` as well, as now it's possible that it'd be called concurrently.

Reply:
Understood, thank you for the review - will be making this change!